app/src/main/res/layout/activity_recommendations.xml

```diff
@@ -31,6 +31,15 @@
                 android:fitsSystemWindows="true"
                 app:layout_collapseMode="parallax" />
 
+            <!-- Text protection scrim. -->
+            <View
```

Comment:
It doesn't look like this scrim is showing up because the height isn't set. You can use the hack in PlannerActivity to set the height of this programmatically (see getStatusBarHeight() and getActionBarHeight()).